roles/openshift_repos/meta/main.yml

```diff
@@ -11,4 +11,5 @@ galaxy_info:
     - 7
   categories:
   - cloud
-dependencies: []
+dependencies:
+- role: openshift_inventory
```

Comment:
Yes it seems fixing https://github.com/openshift/openshift-ansible/pull/3734 hasn't been a priority so we get multiple invocations of role dependencies. That would be nice to get fixed in general, but I don't think it should block this... this role is quick and idempotent (although showing multiple deprecation notices would be pretty annoying, to be sure).